core/cloudflow-operator/src/main/scala/cloudflow/operator/event/ConfigurationScopeLayering.scala

```diff
@@ -132,26 +132,27 @@ object ConfigurationScopeLayering {
                                                     streamletConfig: Config,
                                                     appConfig: Config,
                                                     clusterSecretConfigs: Map[String, Config]): Config = {
+    val defaultClusterConfig = clusterSecretConfigs.get(TopicActions.DefaultConfigurationName)
```

Comment:
Can we have a test, at least checking the `Config` in output contains the expected keys/values?

Reply:
I was having a git issue and forgot to include this. I amended the test in `ConfigurationScopeLayeringSpec` to include the new use case.

Reply:
Nice! Thanks!